code-challenges.test.js

```diff
@@ -34,10 +34,31 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
-
-// Pseudo code:
-
+describe("divisibleOrNah", () => {
+    it(" decides if the number inside it is evenly divisible by three or not", () => {
+        expect(divisibleOrNah(15)).toEqual("15 is divisible by three")
+        expect(divisibleOrNah(0)).toEqual("0 is divisible by three")
+        expect(divisibleOrNah(-7)).toEqual("-7 is not divisible by three")
+    })
+
+})
```

Comment:
The purpose of the test variables is to give you specific input. In this case, you didn't use the test variable in your expect statement which means your code would have falsely passed the test.